administrator/language/en-GB/com_guidedtours.sys.ini

```
@@ -0,0 +1,379 @@
+
+COM_GUIDEDTOURS_TOUR_CONTACTS_STEP_SAVECLOSE_TITLE="Select 'Save and Close'"
+COM_GUIDEDTOURS_TOUR_CONTACTS_STEP_SAVECLOSE_DESCRIPTION="<p>Save and close the contact.</p>"
+
+COM_GUIDEDTOURS_TOUR_CONTACTS_STEP_CONGRATULATIONS_TITLE="Congratulations!"
+COM_GUIDEDTOURS_TOUR_CONTACTS_STEP_CONGRATULATIONS_DESCRIPTION="<p>You have created a contact!</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_TITLE="How to create news feeds?"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_DESCRIPTION="<p>This tour will show you how you can create news feeds in Joomla.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_NEW_TITLE="Select the 'New' button"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_NEW_DESCRIPTION="<p>Select 'New' to create a new news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_TITLE_TITLE="Add a title"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_TITLE_DESCRIPTION="<p>Enter a mandatory title for the news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_ALIAS_TITLE="Enter an alias"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_ALIAS_DESCRIPTION="<p>You can write the internal name of this news feed. You can leave this blank and Joomla will fill a default value in lower case with dashes instead of spaces.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_LINK_TITLE="Enter the link"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_LINK_DESCRIPTION="<p>Add the link leading to the news feed here.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_DESCRIPTION_TITLE="Enter a description"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_DESCRIPTION_DESCRIPTION="<p>Add a description for news feed here.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_STATUS_TITLE="Select the status"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_STATUS_DESCRIPTION="<p>Select the published status of the news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CATEGORY_TITLE="Select a category"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CATEGORY_DESCRIPTION="<p>Select the category for this news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_ACCESS_TITLE="Set the access level"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_ACCESS_DESCRIPTION="<p>Select the access level for the news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_TAGS_TITLE="Add tags"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_TAGS_DESCRIPTION="<p>Select tags for your news feed. You can also enter a new tag by typing the name in the field and pressing enter.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_VERSIONNOTE_TITLE="Add a version note"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_VERSIONNOTE_DESCRIPTION="<p>This is an optional field to identify the version of this news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_SAVECLOSE_TITLE="Select 'Save and Close'"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_SAVECLOSE_DESCRIPTION="<p>Save and close the news feed.</p>"
+
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CONGRATULATIONS_TITLE="Congratulations!"
+COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CONGRATULATIONS_DESCRIPTION="<p>You have created a news feed!</p>"
```

```suggestion
COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CONGRATULATIONS_TITLE="Congratulations!"
COM_GUIDEDTOURS_TOUR_NEWSFEEDS_STEP_CONGRATULATIONS_DESCRIPTION="<p>You have created a news feed!</p>"
```
Maybe remove the second `!` since it is already in the title.

---

Makes sense